packages/delivery-service/src/delivery.ts

```diff
@@ -1,7 +1,7 @@
 import { Acknoledgment, getMessages, schema } from '@dm3-org/dm3-lib-delivery';
 import { auth } from '@dm3-org/dm3-lib-server-side';
 import { validateSchema } from '@dm3-org/dm3-lib-shared';
-import { getConversationId } from '@dm3-org/dm3-lib-storage';
+import { getConversationId } from '@dm3-org/dm3-lib-delivery';
```

Comment:
why has this moved to delivery?

Reply:
It was actually already in dm3-lib-delivery, and also in lib-storage. As the delivery service needs the function, but doesn't use lib storage, I pointed it to lib-delivery. 
Then, afaik, getConversationId in lib-storage was not used anymore, so I removed it.

Reply:
![image](https://github.com/dm3-org/dm3/assets/102788938/4fc8b53e-22ff-4549-b975-d15c0139eae3)
Also, it is only used in ds atm